tests/test_multiworld.py

```diff
@@ -80,3 +80,30 @@ def test_stochastic(self):
         ratio = report['teacher1/exs'].value() / report['teacher2/exs'].value()
         assert ratio > 18
         assert ratio < 22
+
+    def test_with_stream(self):
+        """
+        Test that multi-tasking works with train:stream.
+        """
+        task1 = 'integration_tests:infinite_train'
+        task2 = 'integration_tests:short_fixed'
+        opt = ParlaiParser(True, True).parse_kwargs(
+            task=f'{task1},{task2}',
+            model='fixed_response',
+            fixed_response='Hello!',
+            datatype='train:stream',
+            batchsize=16,
+        )
+        agent = create_agent(opt)
+        world = create_task(opt, agent)
+
+        for i in range(100):
+            world.parley()
+            if i % 10 == 0 and i > 0:
```

Comment:
thank you for the test. could we also inspect the `world.acts` or something to see whether the examples are nontrivial (i.e. not just the default  `{'id':'teacher', 'episode_done': True}` act)

Reply:
done